packages/driver/src/cy/commands/querying/root.ts

```diff
@@ -4,7 +4,7 @@ export default (Commands, Cypress, cy, state) => {
       timeout: options.timeout,
     })
 
-    cy.state('current').set('timeout', options.timeout)
+    this.set('timeout', options.timeout)
```

Comment:
Just to clarify, are there equivalent? My understanding is `cy.state('current')` is just the current Cypress instance. this.set` points to the same thing - an instance of the class defined in `driver/src/cypress.ts`?

Reply:
`cy.state('current')` is the currently running command, not the Cypress instance. Inside queries, looks like `this` is applied as [a command created to wrap the query function](https://github.com/cypress-io/cypress/blob/1cf3c0bffd001a23eaa272220134a892a9c5a814/packages/driver/src/cypress/cy.ts#L854-L866).

Reply:
Yep. This is a DX decision I made around queries - they have direct access to their own `$Command` instance (via `this`), and properties on the command instance are where you store any information that needs to 'make it out' of the query closure (such as timeout).